recipes/pgrc/build.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+mkdir -p $PREFIX/bin
+
+if [[ "$(uname)" == "Linux" ]]; then
+
+mkdir build
+cd build
+export CPATH=${BUILD_PREFIX}/include
+export CXXPATH=${BUILD_PREFIX}/include
+export CFLAGS="$CFLAGS -I$BUILD_PREFIX/include"
```

Comment:
You enhance the `CFLAGS` but later `cmake` does not make use of them via `-DCMAKE_C_FLAGS=${CFLAGS}`